packages/styles/scss/patterns/blocks/feature-card/_feature-card.scss

```diff
@@ -48,6 +49,16 @@
         }
       }
     }
+
+    @include carbon--breakpoint('md') {
+      .#{$prefix}--content-group__title {
+        max-width: 100%;
+      }
+
+      .#{$prefix}--content-group__children {
+        @include hang;
```

Comment:
I'd personally put it on the card itself since that's what's doing the hanging.

Reply:
ok!